src/components/Cards/TestimonialCard/index.tsx

```diff
@@ -1,18 +1,29 @@
 import Image from 'next/image'
 
-import { messageStyle, photoContainerStyle, nameStyle, testimonialCardStyle } from "./styles"
+import {
+  messageStyle,
+  photoContainerStyle,
+  nameStyle,
+  testimonialCardStyle,
+} from './styles'
 
 export interface TestimonialCardProps {
   name: string
   message: string
   image: string | StaticImageData
 }
```

Comment:
Adiciona uma prop `id` para ser utilizado no `Slider`
```suggestion
export interface TestimonialCardProps {
  id?: string
  name: string
  message: string
  image: string | StaticImageData
}
```